Makefile

```diff
@@ -21,8 +22,16 @@ build_local: prepare_bindata
 upload: prepare_upload_data
 	./build_helper.sh upload
 
+go-vet:
+	@go tool vet -all -shadow=true ${PKG}
+
+go-fmt:
+	@gofmt -l $(shell find . -type f -name '*.go')
+
 test: build_local
 	./autospotting
 
 cover:
-	./build_helper.sh calculate_coverage
\ No newline at end of file
+	./build_helper.sh calculate_coverage
+
+.PHONY: all clean prepare_bindata build_lambda_binary prepare_upload_data build_local upload go-vet go-fmt test cover
```

Comment:
I think that you can drop this commit (9e154cf), the first line has a typo, and I'll try to work it out better at some later point in time.